lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -188,6 +188,13 @@ class Server {
     }
 
     function sendRedirect(url) {
+      if (url.split('').some(char => char.charCodeAt(0) > 256)) {
```

Comment:
only semi-related, at one point debugging I put a redirect that contained non-ascii characters and it crashed the server entirely, which is suboptimal. returning a 500 makes more sense :)